iocBoot/iocxxx/softioc/xxx.sh

```diff
@@ -16,6 +16,9 @@
 IOC_NAME=xxx
 # The name of the IOC binary isn't necessarily the same as the name of the IOC
 IOC_BINARY=xxx
+# The absolute path to the directory containing the IOC's bin directory, if running an example IOC binary
+#!IOC_BIN_PARENT=/home/username/epics/synApps/support/xxx
```

Comment:
Should this be IOC_BIN_DIR to match with the other directory definitions?

Reply:
The bin directory has historically been hard-coded and added to the IOC_CMD along with the ${EPICS_HOST_ARCH} subdirectory.  I tried to change as little as possible, which is why I created the unintuitive IOC_BIN_PARENT definition.

I agree it would make more sense to change the variable to IOC_BIN_DIR and then set the IOC_BIN_PATH to be this:
```
    IOC_BIN_PATH="${IOC_BIN_DIR}/${EPICS_HOST_ARCH}/${IOC_BINARY}"
```

Reply:
I changed the variable:

cb599654ba5de64d8963ba58ceeb55e3d21c2dfb